js/app/controllers/content-product.ng

```diff
@@ -138,7 +138,158 @@ mkdir my-new-bucket</pre>
       the Cloud Tools for PowerShell module via <code>Import-Module GoogleCloud</code>.
     </p>
   </div>
+  
+    <!-- Google Cloud BigQuery -->
+  <div ng-switch-when="google-cloud-bigquery">
+    <p>
+      Google BigQuery is a versatile tool that solves the problem of storing and querying massive 
+      datasets without having to worry about data formats, underlying resources, and other things 
+      that distract you from your analysis.
+    </p>
+    
+    <h2>Dataset</h2>
+    <p>
+      To use BigQuery in a Cloud project, first create a <em>Dataset</em> using the 
+      <cmdlet-ref name="New-BqDataset"></cmdlet-ref> cmdlet. This will take in basic information and 
+      create the resource serverside. Locally, a reference Dataset object is returned. To get a reference
+      object for an existing dataset, use <cmdlet-ref name="Get-BqDataset"></cmdlet-ref>.
+    </p>
+    <pre>
+# Makes a new dataset with DatasetId "my_dataset".
+$dataset = New-BqDataset "my_dataset" -Name "Test Dataset" -Description "Some test data for BigQuery"
+
+# The following two lines do the same thing.
+$dataset = Get-BqDataset "my_dataset"
+$dataset = $dataset | Get-BqDataset
+    </pre>
+    <p>
+      This object <em>$dataset</em> can be modified and passed into further cmdlets such as 
+      <cmdlet-ref name="Set-BqDataset"></cmdlet-ref> to manipulate cloud resources. This cmdlet also
+      handles adding and removing labels with <code>-SetLabel</code> and <code>-ClearLabel</code>.
+    </p>
+    <pre>
+# Updates the Name field of $dataset.
+$dataset.Name = "A Different Name"
+$dataset = Set-BqDataset "my_dataset"
 
+# Adds the labels 'test' and 'other' to $dataset.
+$dataset = Set-BqDataset "my_dataset" -SetLabel @{"test"="one";"other"="two"}
+    </pre>
+    <p>
+      Datasets can be deleted by the <cmdlet-ref name="Remove-BqDataset"></cmdlet-ref> cmdlet. This 
+      cmdlet supports ShouldProcess (the <code>-WhatIf</code> parameter) and will prompt for user 
+      confirmation before deleting a non-empty Dataset. This safeguard can be bypassed with the 
+      <code>-Force</code> parameter when scripting.  
+    </p>
+    <pre>
+# Deletes $dataset.
+$dataset | Remove-BqDataset -Force
+    </pre>
+    
+    <h2>Table</h2>
+    <p>
+      Each Dataset has a number of <em>Tables</em> to hold data. Tables are created with the 
+      <cmdlet-ref name="New-BqTable"></cmdlet-ref> cmdlet by passing in a TableId and a Dataset 
```

Comment:
Got it as per Chris's suggestion above.  I just went through the entire doc and threw <code> flags all over the place.  (Where they belonged, that is)